packages/dds/tree/src/test/simple-tree/proxies.spec.ts

```diff
@@ -281,6 +281,25 @@ describe("ArrayNode Proxy", () => {
 			assert.deepEqual(root, [{ id: "A" }, newItem, { id: "C" }]);
 		});
 
+		it("multiple primitives", () => {
+			const root = hydrate(schema, []);
+			assert.deepEqual(root, []);
+			root.insertAt(0, 42, 43);
+			assert.deepEqual(root, [42, 43]);
+		});
+
+		it("multiple objects", () => {
```

Comment:
Should have a test that has a gap between, i.e. our "a...f" scenario.

Reply:
This is covered by the combinatorial tests ("bottom up" variants)